modules/shadertools/tsconfig.json

```diff
@@ -4,6 +4,7 @@
   "exclude": ["node_modules", "test"],
   "compilerOptions": {
     "strict": true,
+    "strictNullChecks": true,
```

Comment:
Minor – I was under the impression that `strictNullChecks` was implied by `strict: true` (https://www.typescriptlang.org/tsconfig#strict). If you'd prefer to be explicit here that's fine with me, but I am curious to understand better if there was some difference!